src/main/java/com/dev/cinema/Main.java

```diff
@@ -16,9 +16,11 @@
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.time.LocalTime;
+import java.util.logging.Logger;
 
 public class Main {
     private static Injector injector = Injector.getInstance("com.dev.cinema");
+    private static Logger logger = Logger.getLogger(Main.class.getName());
```

Comment:
you should be possible to pass class, not a string, after you import the right Logger class